e2e/cypress/integration/plugins/marketplace/marketplace_spec.js

```diff
@@ -300,7 +300,7 @@ describe('Plugin Marketplace', () => {
             cy.get('#marketplace-plugin-github').should('be.visible');
         });
 
-        it('change tab on "install plugins" click', () => {
+        it('change tab on "Installed" click', () => {
```

Comment:
It is actually testing the "Install Plugins" link within the "Installed" tab. Maybe it should be more like something like: 
```suggestion
        it('change tab to "All Plugins" when "Install Plugins" link is clicked', () => {
```

Reply:
Good suggestion :+1: